scrypt/src/params.rs

```diff
@@ -88,7 +88,7 @@ impl Params {
     }
 
     /// Recommended values sufficient for most use-cases
```

Comment:
Perhaps add a link to the OWASP cheat sheet here:

```suggestion
    /// Recommended values according to the [OWASP cheat sheet](https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#scrypt)
```